src/server.ts

```diff
@@ -0,0 +1,6 @@
+import { events, chat, constants } from "resourcemod";
+
+events.onItemPickup((event) => {
+  chat.sayToEveryone(
+    `${event.player.name} just picked up ${event.item}.`, constants.HUD_PRINT_TALK);
```

Comment:
can you make it one line? chat.sayToEveryone(`${event.player.name} just picked up ${event.item}`) smth like that but your code but in one line

Reply:
Sure Prettier actually split it into more i forgot to delete this last one.